cypress/e2e/orders/receiving-and-check-in/receiving-pieces-from-order-for-PE-mix.cy.js

```diff
@@ -0,0 +1,130 @@
+import NewOrganization from '../../../support/fragments/organizations/newOrganization';
+import OrderLines from '../../../support/fragments/orders/orderLines';
+import ItemRecordView from '../../../support/fragments/inventory/itemRecordView';
+import ServicePoints from '../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import NewLocation from '../../../support/fragments/settings/tenant/locations/newLocation';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import ItemActions from '../../../support/fragments/inventory/inventoryItem/itemActions';
+import { ITEM_STATUS_NAMES } from '../../../support/constants';
+import Users from '../../../support/fragments/users/users';
+
+describe('orders: Receiving and Check-in', () => {
+  const order = {
+    ...NewOrder.defaultOneTimeOrder,
+    approved: true,
+  };
+  const organization = {
+    ...NewOrganization.defaultUiOrganizations,
+    accounts: [
+      {
+        accountNo: getRandomPostfix(),
+        accountStatus: 'Active',
+        acqUnitIds: [],
+        appSystemNo: '',
+        description: 'Main library account',
+        libraryCode: 'COB',
+        libraryEdiCode: getRandomPostfix(),
+        name: 'TestAccout1',
+        notes: '',
+        paymentMethod: 'Cash',
+      }
+    ]
+  };
+  const barcodeForFirstItem = Helper.getRandomBarcode();
+  const barcodeForSecondItem = Helper.getRandomBarcode();
+
+
+  let orderNumber;
+  let user;
+  let effectiveLocationServicePoint;
+  let location;
+
+  before(() => {
+    cy.getAdminToken();
+
+    ServicePoints.getViaApi({ limit: 1, query: 'name=="Circ Desk 2"' })
+      .then((servicePoints) => {
+        effectiveLocationServicePoint = servicePoints[0];
+        NewLocation.createViaApi(NewLocation.getDefaultLocation(effectiveLocationServicePoint.id))
+          .then((locationResponse) => {
+            location = locationResponse;
+            Organizations.createOrganizationViaApi(organization)
+              .then(organizationsResponse => {
+                organization.id = organizationsResponse;
+                order.vendor = organizationsResponse;
+              });
+
+            cy.loginAsAdmin({ path:TopMenu.ordersPath, waiter: Orders.waitLoading });
+            cy.createOrderApi(order)
+              .then((response) => {
+                orderNumber = response.body.poNumber;
+                Orders.searchByParameter('PO number', orderNumber);
+                Orders.selectFromResultsList();
+                Orders.createPOLineViaActions();
+                OrderLines.selectRandomInstanceInTitleLookUP('*', 10);
+                OrderLines.fillInPOLineInfoForExportWithLocationForPhisicalResource(`${organization.accounts[0].name} (${organization.accounts[0].accountNo})`, 'Purchase', locationResponse.institutionId, '2');
+                OrderLines.backToEditingOrder();
+              });
+          });
+      });
+
+    cy.createTempUser([
```

Comment:
Once a user is created, it is not deleted at the end of the test. Please fix this

Reply:
Done